src/ovs/aca_vlan_manager.cpp

```diff
@@ -357,4 +357,43 @@ int ACA_Vlan_Manager::get_outports_unsafe(string vpc_id, string &outports)
   return overall_rc;
 }
 
+
+int ACA_Vlan_Manager::get_oam_server_port(string vpc_id, uint32_t *port)
```

Comment:
I would rather to have this function return the udp_port number directly, return 0 if not found.